exercises/saddle-points/canonical-data.json

```
@@ -142,6 +145,11 @@
           "row": 1,
           "column": 1
         }
+      ],
+      "comments": [
+        "If your tests depend on the order of the saddle points,",
+        "note that this case breaks the sorting convention used in other cases (row asc, column asc),",
```

Maybe write out `asc` as `ascending`:

```suggestion
        "note that this case breaks the sorting convention used in other cases (row ascending, column ascencding),",
```